tests/baselines/reference/tsserver/moduleResolution/alternateResult.js

```diff
@@ -998,7 +801,7 @@ Info seq  [hh:mm:ss:mss] event:
               "line": 2,
               "offset": 26
             },
-            "text": "Could not find a declaration file for module 'bar'. '/home/src/projects/project/node_modules/bar/index.mjs' implicitly has an 'any' type.\n  Try `npm i --save-dev @types/bar` if it exists or add a new declaration (.d.ts) file containing `declare module 'bar';`",
```

Comment:
This is the error message given if module resolution fails: and then the message changes based of if we can find the resolution using node10 or not. With not watching this location (the resolution was to ".js" file) we wil not change the error based on whether you later install `@types` which is incorrectly typed